cypress/support/pages/tasks/pageTasks.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+class Task {
+  static #createForm = '[id="homework-form"]'
+  static #taskNameInput = '[data-testid="homework-name"]'
+  static #groupSubmissionCheckbox = '[id="teamSubmissions"]'
+  static #draftCheckbox = '[data-testid="private-checkbox"]'
+  static #visibilityStartDateInput = '[data-testid="form-datetime-input-availableDate"]'
+  static #visibilityDueDateInput = '[data-testid="form-datetime-input-dueDate"]'
+  static #homeWorkDescriptionP = '[class="ck-placeholder"]'
```

Comment:
also data-testid for this element

Reply:
It seems to be an external tool, so we have no possibiility to add ids, so we leave it as it is and adjust later if necessary.